SwiftSampleProject/SwiftyVokoder/Models/Station.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  Station.swift
+//  
+//
+//  Created by Carl Hill-Popper on 11/13/15.
+//  Copyright © 2015 Vokal.
+//
+
+import Foundation
+import CoreData
+import CoreLocation
+import Vokoder
+
+//lets pretend we're using mogenerator to create attribute constants
+enum StationAttributes: String {
+    case name
+    case longitude
+    case locationString
+    case latitude
+    case identifier
+    case descriptiveName
+    case accessible
+}
+
+@objc(Station)
+class Station: NSManagedObject {
+
+    lazy private(set) var coordinate: CLLocationCoordinate2D = {
+        CLLocationCoordinate2D(latitude: Double(self.latitude ?? 0),
+            longitude: Double(self.longitude ?? 0))
+    }()
+}
+
+extension Station: VOKMappableModel {
+    static func coreDataMaps() -> [VOKManagedObjectMap] {
+        return [
+            VOKManagedObjectMap(foreignKeyPath: "MAP_ID",
+                coreDataKeyEnum: StationAttributes.identifier),
+            VOKManagedObjectMap(foreignKeyPath: "STATION_NAME",
+                coreDataKeyEnum: StationAttributes.name),
+            VOKManagedObjectMap(foreignKeyPath: "STATION_DESCRIPTIVE_NAME",
+                coreDataKeyEnum: StationAttributes.descriptiveName),
+            VOKManagedObjectMap(foreignKeyPath: "ADA",
+                coreDataKeyEnum: StationAttributes.accessible),
+            VOKManagedObjectMap(foreignKeyPath: "Location",
+                coreDataKeyEnum: StationAttributes.locationString),
+        ]
+    }
+    
+    static func uniqueKey() -> String? {
+        return StationAttributes.identifier.rawValue
+    }
+    
+    static func importCompletionBlock() -> VOKPostImportBlock {
+        return { (_, inputObject: NSManagedObject) in
```

Comment:
I was opting for clarity by including the type of the params (since this is example code).  I'll add a comment about the `_`.